### PR TITLE
test(hx-file-input): surf-2009

### PR DESCRIPTION
**Pull request by BY00565233**
## Description

unit test `<hx-file-input>`

image

### What are the relevant story cards/tickets? Any additional PRs or other references?

Jira: SURF-2009(unit test case)

## Before you request a review for this PR:

- [x] Did you add component tests for any new code?
- [x] Did you run the component unit tests via `yarn test` to ensure all tests passed?
- [x] Did you include a screenshot of the component tests?
- [x] If you changed/added functionality, did you update the demo page and documentation?
- [x] If needed, did you add or modify the demo test page to test the changed/added functionality?
- [x] Did you assign reviewers?
- [x] In Jira, have you linked to this PR on the ticket(s)?

### CODE REVIEW

File: src/elements/hx-file-input/index.spec.js [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+
+                expect(queryId).to.equal(id);
+            });
+        });
+    });
+
+    describe('test getters and setters', () => {
+        it('should be able to set icon [icon="upload"]', async () => {
+            const component = /** @type {HXFileInputElement} */ await fixture(template);
+
```
Comment (100stacks) [inline src/elements/hx-file-input/index.spec.js:111]: Please remove extra blank line.

Comment (BY00565233) [reply src/elements/hx-file-input/index.spec.js]: Changes have been done, Please review

File: src/elements/hx-file-input/index.spec.js [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+            expect(component).lightDom.to.not.equal(template);
+        });
+
+    });
+
+    describe('test Shadow DOM', () => {
+        describe('verify render', () => {
+            it('should have a static Shadow DOM', async function () {
+                const component = /** @type { HXFileInputElement } */ await fixture(template);
+
```
Comment (100stacks) [inline src/elements/hx-file-input/index.spec.js:39]: Please remove extra line.

File: src/elements/hx-file-input/index.spec.js [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+                const query = shadow.querySelector(elSelector);
+                const queryId = query.id;
+
+                expect(queryId).to.equal(id);
+            });
+        });
+    });
+
+    describe('test getters and setters', () => {
+        it('should be able to set icon [icon="upload"]', async () => {
```
Comment (100stacks) [inline src/elements/hx-file-input/index.spec.js:109]: You are on the right path.  Please update this test to set/check this `[icon="upload"]`.

File: src/elements/hx-file-input/index.spec.js (hunk 2 above)
Comment (BY00565233) [reply src/elements/hx-file-input/index.spec.js]: Changes have been done, Please review

File: src/elements/hx-file-input/index.spec.js (hunk 3 above)
Comment (BY00565233) [reply src/elements/hx-file-input/index.spec.js]: Changes have been done, Please review

File: src/elements/hx-file-input/index.spec.js [hunk 4]
```diff
@@ -0,0 +1,121 @@
...
+            const component = /** @type {HXFileInputElement} */ await fixture(template);
+            const attrType = 'icon';
+
+            component.icon = attrType;
+            const attr = component.hasAttribute('icon');
+            const alertType = component.getAttribute('icon');
+
+            expect(attr).to.be.true;
+            expect(alertType).to.equal(attrType);
+        });
```
Comment (100stacks) [inline src/elements/hx-file-input/index.spec.js:119]: Please revert the wording on this test:

```js
it('should be able to set icon [icon="upload"]', async () => {
```

 Also in its current form, the test is not asserting that the `icon` attribute is equal to `upload`.  Please correct.

Comment (BY00565233) [reply src/elements/hx-file-input/index.spec.js]: Changes have been done, Please review

**Pull request merged**